functionary/ui/views/builds.py

```diff
@@ -0,0 +1,20 @@
+from builder.models import Build, BuildLog
+
+from .view_base import (
+    PermissionedEnvironmentDetailView,
+    PermissionedEnvironmentListView,
+)
+
+
+class BuildListView(PermissionedEnvironmentListView):
+    model = Build
+    order_by_fields = ["id"]
```

Comment:
Because we're using uuids, sorting by id ends up not being especially meaningful.  I think we want this to instead by:

```python
order_by_fields = ["-created_at"]
```